.github/workflows/cr.yml

```diff
@@ -0,0 +1,25 @@
+name: Continuous Releases
+
+on: [push, pull_request]
```

Comment:
Just stumbled across this... The recommended setup is on approval only: https://github.com/stackblitz-labs/pkg.pr.new?tab=readme-ov-file#release-approved-pull-requests-only

Reply:
Yea, exactly, but this approach has no issues at all if fast feedback is more appealing!